### ISSUE TITLE
Displaced lobar tracheal bronchus

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6631]
**Preferred term label:** Displaced lobar tracheal bronchus

**Synonyms**
'Pig bronchus'
'Bronchus suis'

**Definition (free text, please give PubMed ID)**
Accessory entire right upper lobe bronchial system originating from the trachea with absent anatomically normal upper lobe bronchus. PMID: 11158647, PMID: 19332762

**Parent term (use hpo.jax.org/app)**
Tracheal bronchus (once created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
